test/test-esm.mjs

```diff
@@ -0,0 +1,34 @@
+// Packages
+import test from 'node:test';
+import assert from 'node:assert';
+
+// Utilities
+import {UIDCHARS} from '../lib/esm/chars';
+import {uid} from '../lib/esm/index';
```

Comment:
Pretty sure these need the file extension to work with ESM

Reply:
Yeah, and I had to change the source code as well to include file extensions in order for the ESM test to work without `--es-module-specifier-resolution=node`